Lil-Peep/src/styles/modules/featured-kits.scss

```diff
@@ -0,0 +1,383 @@
+.featured-kits {
+  font-family: 'Oswald', sans-serif;
+
+  .big-banner {
+    background-position: top;
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+}
+
+.big-banner-content {
+  background: linear-gradient(0deg,
+    rgb(251,239,227),
+    rgba(251,239,227, 1) 20%,
+    rgba(251,239,227, 0.2) 50%,
+    rgba(251,239,227, 0.1));
+  display: flex;
+  flex-direction: column;
+  height: 320px;
+  justify-content: flex-end;
+  text-align: center;
+
+  .big-banner-title {
+    font-size: 20px;
+    margin-bottom: 0;
+  }
+
+  .big-banner-description {
+    font-family: 'Lato', sans-serif;
+    font-size: 12px;
+    margin: 0;
+  }
+
+  .view-in-detail {
+    background-color: #4a90e2;
+    border-radius: 3px;;
```

Comment:
Declaration should be terminated by a single semicolon

Reply:
Check this out! :wink: